met-web/src/components/engagement/form/EngagementFormTabs/Links/PublicUrls.tsx

```diff
@@ -140,10 +140,34 @@ export const PublicUrls = () => {
                             }}
                             InputProps={{
                                 startAdornment: (
-                                    <InputAdornment position="start" sx={{ height: '100%', maxHeight: '100%' }}>
```

Comment:
when trying to fix the font weight for the secondary button the text color is different between both I changed it to text field to match the dashboard url input adornment